plugin/src/main/java/com/denizenscript/denizen/events/block/BellRingScriptEvent.java

```diff
@@ -0,0 +1,73 @@
+package com.denizenscript.denizen.events.block;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BellRingEvent;
+
+public class BellRingScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // bell rings
+    //
+    // @Location true
+    //
+    // @Group Block
+    //
+    // @Cancellable true
+    //
+    // @Triggers when a bell block rings. (Requires Paper on versions lower than 1.19)
+    //
+    // @Context
+    // <context.entity> returns the EntityTag that rung the bell, if any.
+    // <context.location> returns the LocationTag of the bell being rung.
+    // <context.direction> returns the ElementTag of the direction the bell was rung. Available only on MC 1.19+.
```

Comment:
Should tell you what can the directions be (probably just link <https://hub.spigotmc.org/javadocs/spigot/org/bukkit/block/BlockFace.html>)
Also I think `direction the bell was rung in` or `in which the bell was rung` might be better here?